### ISSUE TITLE
Type error for `_request()`

### ISSUE THREAD

**Issue by imguangyu** [vaseline555/Federated-Learning-in-PyTorch#13]
Thanks for the well-organized code base. However, when I run the example experiments, it shows that two arguments are missing. Are the default values of them missing? 
```
TypeError: _request() missing 2 required positional arguments: 'participated' and 'retain_model'
```

**Comment by zr-wang03** [comment]
Same issue here.

**Comment by vaseline555** [comment]
Oops, I thought some dev code was not properly transplanted.
Fixed in commit bb6ca682ee63039647caa1dd5d7983c3d296adfc.
Please check and tell me if other errors are observed.

Thank you for raising this issue!
고마워요! @imguangyu @zr-wang03
